### ISSUE TITLE
🐛 Long skill family chips look pretty bad

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#10037]
## 🐛 Bug

Describe the issue you encountered.

## 📋 Steps to Reproduce

1. http://localhost:8000/en/admin/settings/skills
2. Narrow viewport to about 1230px with sidemenu open

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/db4686b2-aaa9-408b-93e2-8b30b9c6d86d)

## 🙋‍♀️ Proposed Solution

Center align text?

## ✅ Acceptance Criteria

- [ ] Long skill family chips look better

**Comment by gobyrne** [comment]
@substrae the padding on these was recently reduced and it's having some unintended consequences (see screenshot). Could you look at bumping it up again or addressing this in another way?

**Comment by mnigh** [comment]
> This doesn't have to do with the padding changes, they were always horrible. To be honest, the solution here is to not use `Pill`/`Chip` elements, because this isn't their intended use case. I think this information was put into them under the assumption that it visually differentiated them in the table, but IMO they should just be a list, or if only ever a single item, plain text.

If it the skill families name were simple titles without punctuation, then a comma separated list would probably be the easy solution for this case. Unfortunately, they do use punctuation.

Examples of skill families that use either commas or a hyphen:

- Programming Languages, Frameworks, and Tools
- Software Solutions - Quality Assurance

## Possible solutions

1. Replace `Chip` with `span` and use a semicolon as a separator (if multiple items) and a period at the end of the list.
2. Replace `Chip` with `span` and use another character as a separator (if multiple items)

**Comment by substrae** [comment]
@mnigh I was thinking more along the lines of a semantic list using `<ul>`.

**Comment by mnigh** [comment]
> I was thinking more along the lines of a semantic list using `<ul>`.

@substrae 🆒 Would you want a check to only use an ordered list if there are multiple skill families? Also, I wonder if we could hide the `description` column by default, it would provide a much better display for most users.

## Screenshots

### Description column shown
<img width="1280" alt="Screen Shot 2024-05-08 at 16 20 38" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/ebde1589-942f-46da-a6b1-afe85a80537b">

### Description column hidden
<img width="1296" alt="Screen Shot 2024-05-08 at 16 20 48" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/26e32c5c-0761-4033-874a-5d2f5512f35a">

**Comment by substrae** [comment]
@mnigh
* RE removing the list semantics for a single item - I'm actually unsure where I fall on this; I know in the past we've debated the [lack of] value around single item lists not being lists, but there's an argument to be made for the information a semantic list communicates, namely that the item exists in a context where a list is possible - this reinforces the idea that the item is a single item in a context where multiple items are possible. At the end of the day I'll defer to @esizer here.
* In terms of hiding the `Description` column - I totally see what you mean in the screenshot, but I'm not the one to make a call there (@Jerryescandon). I imagine there's probably some kind of value to having it shown, but it's equally possible we've discovered it's not necessary. An alternative solution might be to narrow the `Description` column a bit, giving the `Skill families` column more horizontal space.

**Comment by esizer** [comment]
> * RE removing the list semantics for a single item - I'm actually unsure where I fall on this; I know in the past we've debated the [lack of] value around single item lists not being lists, but there's an argument to be made for the information a semantic list communicates, namely that the item exists in a context where a list is possible - this reinforces the idea that the item is a single item in a context where multiple items are possible. At the end of the day I'll defer to @esizer here.

Yeah, when it is a dynamic list where we don't know how many will appear it seems fine. I'm mostly concerned about not using lists when we _know_ there only ever will be one item.

**Comment by Jerryescandon** [comment]
We could do two things to improve the amount of space the description uses.
- First we could reduce the size to be the width to be the same as other columns.
- Secondly we could truncate the text the way the "notes" work on other tables. This way the user can expand it if they want to read more about it.
